shuup/core/models/_orders.py

```diff
@@ -141,7 +143,7 @@ def get_default_complete(self):
 
 
 @python_2_unicode_compatible
-class OrderStatus(TranslatableModel):
+class OrderStatus(TranslatableModel, models.Model):
```

Comment:
why you need to add this model, it is already being used in `TranslatableModel`?